src/VsixTesting.Xunit/Internal/VsTestCaseFactory.cs

```diff
@@ -80,18 +85,26 @@ internal static IEnumerable<Instance> GetInstances(IEnumerable<VsInstallation> i
 
         internal static IEnumerable<VsInstallation> FilterInstallations(IEnumerable<VsInstallation> installations, VsTestSettings settings, StringBuilder output = null)
         {
+            var parentVsInstallation = ParentVsProcess.Value == null ? null
+                : installations.FirstOrDefault(i => i.ApplicationPath.Equals(ParentVsProcess.Value.MainModule.FileName, StringComparison.OrdinalIgnoreCase));
+
             foreach (var group in installations.GroupBy(i => i.Version.Major).OrderBy(g => g.Key))
             {
-                var installation = group.OrderBy(i => i.Name.Contains("-pre")).First();
-
-                if (!settings.SupportedVersionRanges.Any(range => installation.Version >= range.Minimum && installation.Version <= range.Maximum))
+                foreach (var installation in group.OrderBy(i => i.Name.Contains("-pre")))
```

Comment:
I thought we could give a priority boost for installations that are the same as the parent process.

Something like this:
```suggestion
                foreach (var installation in group
                    .OrderBy(i => !ParentVsProcess.Value?.MainModule.FileName.Equals(i.Path, StringComparison.OrdinalIgnoreCase))
                    .ThenBy(i => i.Name.Contains("-pre")))
```

Reply:
Your implementation seems to make more sense but do you understand that both end up giving the same behavior right?

Reply:
I thought it was returning the `parentVsInstallation`, even when a different version was installed and within the target range (e.g. it wasn't testing with 2015 when I was running using 2017).